tests/test_main.py

```
@@ -607,6 +607,205 @@ def _capture_create_task(coro: Any, *args: Any, **kwargs: Any) -> Any:
+    @patch("mmrelay.main.start_message_queue")
+    @patch("mmrelay.main.connect_meshtastic")
+    @patch("mmrelay.main.connect_matrix", new_callable=AsyncMock)
+    @patch("mmrelay.main.join_matrix_room", new_callable=AsyncMock)
+    @patch(
+        "mmrelay.main.meshtastic_utils.refresh_node_name_tables", new_callable=AsyncMock
+    )
+    @patch("mmrelay.main.stop_message_queue")
+    def test_main_publishes_ready_after_sync_start_and_drain_completion(
+        self,
+        mock_stop_queue,
+        mock_refresh_node_names,
```

Ruff (0.15.9)

[warning] Unused method argument: `mock_refresh_node_names`

(ARG002)